components/raftstore/src/store/peer.rs

```diff
@@ -3205,6 +3211,31 @@ where
                 };
             }
         }
+        let flags = WriteBatchFlags::from_bits_check(req.get_header().get_flags());
+        if flags.contains(WriteBatchFlags::STALE_READ) {
+            let read_ts = decode_u64(&mut req.mut_header().take_flag_data().as_slice()).unwrap();
```

Comment:
I think you can write it as `&mut req.get_header().get_flag_data()`.

Reply:
Why mark as resolved?

Reply:
Oh, there is one more place, it is changed now.